src/GafferSceneUI/TranslateTool.cpp

```diff
@@ -116,6 +116,12 @@ void TranslateTool::updateHandles()
 {
 	handles()->setTransform( handlesTransform() );
 
+	// Because we provide multiple orientations, the handles
+	// may well not be aligned with the axes of the transform
+	// space. So any given handle might affect several components
+	// of the target translation. For each handle, check to see
+	// if each of the plugs it effects are settable, and if not,
+	// disable the handle.
```

Comment:
> It kind of sounds like the enabled state of the handle should be taking the orientation plug into account?

I think you figured this out, but that's exactly what is happening already.

> Maybe having the orientation default to World would have avoided the confusion on my end.

Does this have the potential to add confusion when you've got a non-rotated cube with a locked translate axis inside a rotated group?

```
import Gaffer
import GafferScene
import IECore

Gaffer.Metadata.registerNodeValue( parent, "serialiser:milestoneVersion", 0, persistent=False )
Gaffer.Metadata.registerNodeValue( parent, "serialiser:majorVersion", 31, persistent=False )
Gaffer.Metadata.registerNodeValue( parent, "serialiser:minorVersion", 0, persistent=False )
Gaffer.Metadata.registerNodeValue( parent, "serialiser:patchVersion", 0, persistent=False )

__children = {}

__children["Cube"] = GafferScene.Cube( "Cube" )
parent.addChild( __children["Cube"] )
__children["Cube"].addChild( Gaffer.V2fPlug( "__uiPosition", defaultValue = IECore.V2f( 0, 0 ), flags = Gaffer.Plug.Flags.Default | Gaffer.Plug.Flags.Dynamic, ) )
__children["Cube"]["__uiPosition"].setValue( IECore.V2f( -11.3500004, 3.84999895 ) )
__children["Group"] = GafferScene.Group( "Group" )
parent.addChild( __children["Group"] )
__children["Group"]["in"].addChild( GafferScene.ScenePlug( "in1", flags = Gaffer.Plug.Flags.Default | Gaffer.Plug.Flags.Dynamic, ) )
__children["Group"]["transform"]["rotate"].setValue( IECore.V3f( 0, 45, 0 ) )
__children["Group"].addChild( Gaffer.V2fPlug( "__uiPosition", defaultValue = IECore.V2f( 0, 0 ), flags = Gaffer.Plug.Flags.Default | Gaffer.Plug.Flags.Dynamic, ) )
__children["Group"]["__uiPosition"].setValue( IECore.V2f( -9.85084534, -5.31406307 ) )
Gaffer.Metadata.registerPlugValue( __children["Cube"]["transform"]["translate"]["x"], 'readOnly', True )
__children["Group"]["in"]["in0"].setInput( __children["Cube"]["out"] )


del __children
```

Maybe the default should be Parent, since it improves both situations?